engine/crates/engine-v2/src/execution/mod.rs

```diff
@@ -89,10 +90,10 @@ pub(crate) struct QueryModifications {
 }
 
 // Modifies the response based on a given rule
-#[allow(unused)]
 pub(crate) struct ResponseModifierExecutor {
     pub rule: ResponseModifierRule,
     /// Which object & fields are impacted
+    /// ordered
```

Comment:
ordered by what?

Reply:
should have said `sorted`, it's just sorted

Reply:
By what though?  Or just the "natural" order that a tuple of ints has?

Reply:
yeah just the natural order

Reply:
just `sort_unstable()`

Reply:
Maybe worth clarifying that in the comment?  It wasn't immediately obvious for some reason (guess maybe it should have been, but I dunno)

Reply:
`sorted by natural order`?

Reply:
works for me